### PR TITLE
google: Fix error caused by subsequent inits.

### PR DESCRIPTION
**Pull request by tejasmanohar**
If a user who doesn't have GCE setup locally init'd the cluster
twice, the error wouldn't be returned by the client init function
the second time since we run the block that causes an error in
once.Do. This removes the clever code in favor of the simpler,
correct solution.

### CODE REVIEW

File: cluster/google/client.go [hunk 1]
```diff
@@ -4,12 +4,10 @@ import (
 	"context"
```
Comment (kklin) [inline cluster/google/client.go]: Commit message shouldn't have trailing period

File: cluster/google/client.go [hunk 2]
```diff
@@ -37,16 +35,13 @@ type clientImpl struct {
...
-
-		service, err = newComputeService(context.Background())
-	})
+	keyfile := filepath.Join(os.Getenv("HOME"), ".gce", "quilt.json")
+	err := os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", keyfile)
+	if err != nil {
+		panic(err)
+	}
+
+	service, err = newComputeService(context.Background())
```
Comment (kklin) [inline cluster/google/client.go]: Doesn't have to be this PR, but does `service` have to be a global variable?

Comment (tejasmanohar) [inline cluster/google/client.go]: ya fixed

File: cluster/google/client.go (hunk 1 above)
Comment (tejasmanohar) [inline cluster/google/client.go]: fixed

**Comment by tejasmanohar** [comment]
ready :)

**Comment by kklin** [comment]
Thanks @tejasmanohar, will merge after travis is happy

**Pull request merged**